### DISCUSSION TITLE
[DE] We can't ask Home Assistant where a person is

### DISCUSSION THREAD

**Discussion by DeepCoreSystem** [OHF-Voice/intents#2231] [_Q&A]
This is the result I get when using the sentence parser in the Developer tools:
![grafik](https://github.com/home-assistant/intents/assets/3703938/3f5600c5-3754-4915-acbe-46c4f709048b)

I found that the file `sentences/en/person_HassGetState.yaml` is missing in the german folder.

So far I managed to create a GitHub CodeSpace ... but I don't really know how to go about it now to create the missing file for german language.

Should I just create a new file `sentences/de/person_HassGetState.yaml`, copy over the content from the english file and translate to german?

And after I can create a Pull Request?

**Comment by tetele** [comment]
It won't work. Check out this discussion https://github.com/home-assistant/intents/discussions/1768

**Reply by DeepCoreSystem** [reply]
I don't understand #1768 ... the `person_HassGetState.yaml` does exist in french !

So it should be possible to do the very same in german, shouldn't it?

I am a native speaker of German and French

**Reply by tetele** [reply]
It exists in French, but it doesn't work in English, French, German or any other language. The discussion I pointed to explains why. The fix I proposed was not accepted, therefore, the problem persists.

There's actually [a contribution](https://github.com/home-assistant/intents/pull/1777) for the German language, but it was only merged to the branch discussed in #1768, not to `main`.
